model/credentials/db.go

```diff
@@ -0,0 +1,200 @@
+package credentials
+
+import (
+	"context"
+	"time"
+
+	"github.com/evergreen-ci/certdepot"
+	"github.com/evergreen-ci/evergreen"
+	"github.com/mongodb/grip"
+	"github.com/mongodb/jasper/rpc"
+	"github.com/pkg/errors"
+	"github.com/square/certstrap/depot"
+)
+
+const (
+	Collection  = "credentials"
+	CAName      = "evergreen"
+	ServiceName = "evergreen.mongodb.com"
```

Comment:
We should take into account that people might operate Evergreen with a different domain name. These values should come from the global environment.

Reply:
I'd hang this off of evergreen.Settings, but yes.

Reply:
I added `DomainName` to `evergreen.Settings`. If the domain name ever changes, it will generate new evergreen credentials that it associates with the new domain name.